instrumentation/pekko/pekko-http-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pekkohttp/v1_0/server/route/PekkoRouteHolder.java

```diff
@@ -10,18 +10,17 @@
 import io.opentelemetry.context.Context;
 import io.opentelemetry.context.ContextKey;
 import io.opentelemetry.context.ImplicitContextKeyed;
-import io.opentelemetry.instrumentation.api.semconv.http.HttpServerRoute;
-import io.opentelemetry.instrumentation.api.semconv.http.HttpServerRouteSource;
-import java.util.ArrayDeque;
 import java.util.Deque;
+import java.util.LinkedList;
+import org.apache.pekko.http.scaladsl.model.Uri;
 
 public class PekkoRouteHolder implements ImplicitContextKeyed {
-  private static final ContextKey<PekkoRouteHolder> KEY = named("opentelemetry-pekko-route");
+  public static final ContextKey<PekkoRouteHolder> KEY = named("opentelemetry-pekko-route");
```

Comment:
instead of making the key public you could add a static method that returns `PekkoRouteHolder ` from `Context`